test/server/guarddog_impl_test.cc

```diff
@@ -24,21 +24,26 @@ namespace Server {
 class GuardDogDeathTest : public testing::Test {
 protected:
   GuardDogDeathTest()
-      : config_kill_(1000, 1000, 100, 1000), config_multikill_(1000, 1000, 1000, 500),
-        time_point_(std::chrono::system_clock::now()) {}
+      : config_kill_(1000, 1000, 100, 1000), config_multikill_(1000, 1000, 1000, 500) {}
+
+  void SetupMockTimeSource() {
+    ON_CALL(time_source_, currentSystemTime())
```

Comment:
Just put this directly in constructor, then you don't have to call from each test.